source/elements/oneMKL/source/domains/spblas/operations/spmv.rst

```diff
@@ -0,0 +1,378 @@
+
+   spmv_descr
+      Descriptor initialized with ``init_spmv_descr``.
+
+.. container:: section
+
+   .. rubric:: Throws
+
+   This function shall throw the following exceptions if the associated
+   condition is detected. An implementation may throw additional
+   implementation-specific exception(s) in case of error conditions not covered
+   here.
+
+   | :ref:`oneapi::mkl::invalid_argument<onemkl_exception_invalid_argument>`
+   | :ref:`oneapi::mkl::unimplemented<onemkl_exception_unimplemented>`
+   | :ref:`oneapi::mkl::unsupported_device<onemkl_exception_unsupported_device>`
+
+.. _onemkl_sparse_spmv_alg:
+
+spmv_alg
+--------
+
+.. rubric:: Syntax
+
+.. code-block:: cpp
+
+   namespace oneapi::mkl::sparse {
+
+       enum class spmv_alg {
+           default_alg,
+           coo_alg1,
+           coo_alg2,
+           csr_alg1,
+           csr_alg2,
+           csr_alg3,
+       };
+
+   }
+
+.. list-table::
+   :header-rows: 1
+   :widths: 10 30 45
+
+   * - Value
+     - Description
+     - Backend equivalent
+   * - ``default_alg``
+     - Default algorithm.
+     - | MKL: N/A
+       | cuSPARSE: ``CUSPARSE_SPMV_ALG_DEFAULT``
+       | rocSPARSE: ``rocsparse_spmv_alg_default``
+   * - ``coo_alg1``
+     - Default algorithm for COO format.
+     - | MKL: N/A
+       | cuSPARSE: ``CUSPARSE_SPMV_COO_ALG1``
+       | rocSPARSE: ``rocsparse_spmv_alg_coo``
+   * - ``coo_alg2``
+     - Deterministic algorithm for COO format.
+     - | MKL: N/A
+       | cuSPARSE: ``CUSPARSE_SPMV_COO_ALG2``
+       | rocSPARSE: ``rocsparse_spmv_alg_coo_atomic``
+   * - ``csr_alg1``
+     - Default algorithm for CSR format.
+     - | MKL: N/A
+       | cuSPARSE: ``CUSPARSE_SPMV_CSR_ALG1``
+       | rocSPARSE: ``rocsparse_spmv_alg_csr_adaptive``
+   * - ``csr_alg2``
+     - Deterministic algorithm for CSR format.
+     - | MKL: N/A
+       | cuSPARSE: ``CUSPARSE_SPMV_CSR_ALG2``
+       | rocSPARSE: ``rocsparse_spmv_alg_csr_stream``
+   * - ``csr_alg3``
+     - LRB variant of the algorithm for CSR format.
+     - | MKL: N/A
+       | cuSPARSE: N/A
+       | rocSPARSE: ``rocsparse_spmv_alg_csr_lrb``
+
+.. _onemkl_sparse_spmv:
+
+spmv
+----
+
+.. rubric:: Syntax
+
+.. code-block:: cpp
+
+   namespace oneapi::mkl::sparse {
+
+       void spmv_buffer_size(
+           sycl::queue                                &queue,
+           oneapi::mkl::transpose                     opA,
+           void*                                      alpha,
+           oneapi::mkl::sparse::matrix_view           A_view,
+           oneapi::mkl::sparse::matrix_handle_t       A_handle,
+           oneapi::mkl::sparse::dense_vector_handle_t x_handle,
+           void*                                      beta,
+           oneapi::mkl::sparse::dense_vector_handle_t y_handle,
+           oneapi::mkl::sparse::spmv_alg              alg,
+           oneapi::mkl::sparse::spmv_descr_t          spmv_descr,
+           std::size_t                                &temp_buffer_size,
```

Comment:
Please remove the commas in after the last arguments of all the APIs.

Reply:
Done in https://github.com/oneapi-src/oneAPI-spec/pull/522/commits/c22684f57c4b8758be4fd877c0dea999bc248b86